### PR TITLE
#6 Project - HTML Responsive Design (The Next Web Clone)

### PR DESCRIPTION
**Pull request by sergioobolevich**
In this project, I practiced my skills in responsive design using just media queries in CSS.
I did this project using flexbox, grid, and floats to align the divs and change the grid template in each screen size.

- Media queries for desktop, tablet an cellphone
- Header responsiveness using flex and float for the logo
- Sections made using grid layout
- Footer using flexbox

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,637 @@
...
+  }
+}
+
+/* END of Mobile Screen Style */
+
+/* TABLET - Styles for Tablet Screen */
+@media (min-width: 768px) and (max-width: 1024px) {
+  /* .logo {
+    display: inline-flex;
+  } */
```
Comment (SpaYco) [inline css/style.css]: - [x] Remove Commented Code ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: index.html [hunk 2]
```diff
@@ -0,0 +1,798 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="initial-scale=1.0" />
```
Comment (SpaYco) [inline index.html]: - [x] Fix the linting error `The 'viewport' meta element 'content' attribute value should contain 'width=device-width'.`

add `width=device-width` to the content attribute.

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [x] fix the linting issues

- [x] The Navbar should always be on the screen even when you scroll down ([link to site](https://thenextweb.com/))

- [x] all the big images are not taking the full width on all screens, do not specify an exact pixel number, instead make it responsive

example: 
image


- [x] The **Latest Funding Round** cards should change from 4 to 3 on medium and small screens

- [x] on medium screens The 3 small titles next to **Latest Funding Round** should become under it in one line, yours is on top of it

**Yours vs. Original** 

image

- [x] on small screens the small titles next to **Latest Funding Round** should be under it, each link on its own line

**Yours vs. Original**

image


- [x] the big 3 images and latest news section merge on medium screens

image


- [x] The Categories Section should be 2 rows on medium screens 

**Yours Vs. Original**

image


- [x] You have a horizontal overflow on specific widths

image

image

image

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️


#### Hey @chubaquelo, its always nice to review your projects! I must say that overall the project looks really nice and I can see that you made some required changes, and don't worry, you're almost done, there are just some required changes that you have to implement in order to get approved, and also add a nice project to your portfolio:


_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.

## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
### Large screens issues:
- [x] Add a gap between those articles (Check image below)
- [x] Add the TNW logo at your footer (Check image below)
### Yours
image
### Original
image

### Medium screen issues
- [x] Those images should fill the whole width of its content (So the text will remain below the image, not beside). Latest deals is also facing the same issue.(Check image below)
#### Yours
image
#### Original
image

- [x] Your latest section with the class `.cat-last-art-latest` (selected⤵) should end at the same place that that image above it. (Check image below)
#### Yours
image
#### Original
image

### Small screen issues
- [x] There are too much margin at the x-axis, it would be better if you decrease it. (Check image below)
#### Yours
image
#### Original
image

- [x] Your footer is breaking (all items should stay in the same line) at small screens, kindly fix that. (Check the image below)
- [x] You have a blank space at the end of your page on small screens. Please get rid of that. (Check the image below)
#### Yours
image
#### Original
image


## [OPTIONAL CHANGES]
**⚠These are mandatory changes, but since It wasn't pointed out earlier, I'll mark them as [OPTIONAL]:**
#### Good practices (Those changes to make you code like a pro 👽)⤵
- [ ] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image

### Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you at the top🚀

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question about this review? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

## Congratulations @chubaquelo! Good job making my previously requested changes, you have also followed the best practices throughout the code, you got my attention and have my approval to move on!_ 🥇💯

### It was a pleasure to review your code


## As always, I wish you all the very best and see you at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**